pkg/datasource/listServices.go

```diff
@@ -16,6 +18,21 @@ type ListServicesQueryData struct {
 	IncludeLinkedAccounts bool   `json:"includeLinkedAccounts,omitempty"`
 }
 
+func buildKeyAttributes(keyAttributes map[string]string) string {
```

Comment:
Can this be done with something like json marshal instead and return the string result of the marshalled json? Thinking of cases where an attribute might need to be escaped and building it this way may lead to a malformed json string.